diff --git a/dist/node/serve-CEEKiiuH.js b/dist/node/serve-CEEKiiuH.js
index 6fdb5a336b38b8e94dbb8c7a86ed5c2756c5bd0a..4433d13b490e984e01f1e3b7f19eadc5357e6c1f 100644
--- a/dist/node/serve-CEEKiiuH.js
+++ b/dist/node/serve-CEEKiiuH.js
@@ -36959,11 +36959,12 @@ function preWrapperPlugin(md, options) {
   md.renderer.rules.fence = (...args) => {
     const [tokens, idx] = args;
     const token = tokens[idx];
+    const title = extractTitle(token.info);
     token.info = token.info.replace(/\[.*\]/, "");
     const active = / active( |$)/.test(token.info) ? " active" : "";
     token.info = token.info.replace(/ active$/, "").replace(/ active /, " ");
     const lang = extractLang(token.info);
-    return `<div class="language-${lang}${getAdaptiveThemeMarker(options)}${active}"><button title="${options.codeCopyButtonTitle}" class="copy"></button><span class="lang">${lang}</span>` + fence(...args) + "</div>";
+    return `<div title="${title}" class="language-${lang}${getAdaptiveThemeMarker(options)}${active}"><button title="${options.codeCopyButtonTitle}" class="copy"></button><span class="lang">${lang}</span>` + fence(...args) + "</div>";
   };
 }
 function getAdaptiveThemeMarker(options) {
diff --git a/types/default-theme.d.ts b/types/default-theme.d.ts
index 58d7d89ff6e77783a54e8f256d12d6e9d82eba26..e89e4c3b5c7c77b72678d96ed6e9a09e8236395a 100644
--- a/types/default-theme.d.ts
+++ b/types/default-theme.d.ts
@@ -127,8 +127,8 @@ export namespace DefaultTheme {
     langMenuLabel?: string
 
     search?:
-      | { provider: 'local'; options?: LocalSearchOptions }
-      | { provider: 'algolia'; options: AlgoliaSearchOptions }
+    | { provider: 'local'; options?: LocalSearchOptions }
+    | { provider: 'algolia'; options: AlgoliaSearchOptions }
 
     /**
      * @deprecated Use `search` instead.
@@ -199,26 +199,26 @@ export namespace DefaultTheme {
 
   export type ThemeableImage =
     | string
-    | { src: string; alt?: string; [prop: string]: any }
-    | { light: string; dark: string; alt?: string; [prop: string]: any }
+    | { src: string; alt?: string;[prop: string]: any }
+    | { light: string; dark: string; alt?: string;[prop: string]: any }
 
   export type FeatureIcon =
     | string
     | {
-        src: string
-        alt?: string
-        width?: string
-        height?: string
-        wrap?: boolean
-      }
+      src: string
+      alt?: string
+      width?: string
+      height?: string
+      wrap?: boolean
+    }
     | {
-        light: string
-        dark: string
-        alt?: string
-        width?: string
-        height?: string
-        wrap?: boolean
-      }
+      light: string
+      dark: string
+      alt?: string
+      width?: string
+      height?: string
+      wrap?: boolean
+    }
 
   // sidebar -------------------------------------------------------------------
 
@@ -263,6 +263,11 @@ export namespace DefaultTheme {
      */
     docFooterText?: string
 
+    /**
+     * Icon for the item
+     */
+    icon?: string
+
     rel?: string
     target?: string
   }
