diff --git a/dist/node/serve-C_2bYS0g.js b/dist/node/serve-C_2bYS0g.js
index f08d8913720be84d4b138e46a543505495e381e9..0c7eec73191dabfa64509156ceb0385519eadc20 100644
--- a/dist/node/serve-C_2bYS0g.js
+++ b/dist/node/serve-C_2bYS0g.js
@@ -36818,12 +36818,13 @@ function preWrapperPlugin(md, options) {
   md.renderer.rules.fence = (...args) => {
     const [tokens, idx] = args;
     const token = tokens[idx];
+    const title = extractTitle(token.info);
     token.info = token.info.replace(/\[.*\]/, "");
     const active = / active( |$)/.test(token.info) ? " active" : "";
     token.info = token.info.replace(/ active$/, "").replace(/ active /, " ");
     const lang = extractLang(token.info);
     const rawCode = fence(...args);
-    return `<div class="language-${lang}${getAdaptiveThemeMarker(
+    return `<div title="${title}" class="language-${lang}${getAdaptiveThemeMarker(
       options
     )}${active}"><button title="Copy Code" class="copy"></button><span class="lang">${lang}</span>${rawCode}</div>`;
   };
