diff --git a/dist/node/serve-CEEKiiuH.js b/dist/node/serve-CEEKiiuH.js
index 6fdb5a336b38b8e94dbb8c7a86ed5c2756c5bd0a..4433d13b490e984e01f1e3b7f19eadc5357e6c1f 100644
--- a/dist/node/serve-CEEKiiuH.js
+++ b/dist/node/serve-CEEKiiuH.js
@@ -36959,11 +36959,12 @@ function preWrapperPlugin(md, options) {
   md.renderer.rules.fence = (...args) => {
     const [tokens, idx] = args;
     const token = tokens[idx];
+    const title = extractTitle(token.info);
     token.info = token.info.replace(/\[.*\]/, "");
     const active = / active( |$)/.test(token.info) ? " active" : "";
     token.info = token.info.replace(/ active$/, "").replace(/ active /, " ");
     const lang = extractLang(token.info);
-    return `<div class="language-${lang}${getAdaptiveThemeMarker(options)}${active}"><button title="${options.codeCopyButtonTitle}" class="copy"></button><span class="lang">${lang}</span>` + fence(...args) + "</div>";
+    return `<div title="${title}" class="language-${lang}${getAdaptiveThemeMarker(options)}${active}"><button title="${options.codeCopyButtonTitle}" class="copy"></button><span class="lang">${lang}</span>` + fence(...args) + "</div>";
   };
 }
 function getAdaptiveThemeMarker(options) {
